std/experimental/allocator/building_blocks/scoped_allocator.d

```diff
@@ -86,8 +86,9 @@ struct ScopedAllocator(ParentAllocator)
         auto b = parent.allocate(n);
         if (!b.ptr) return b;
         Node* toInsert = & parent.prefix(b);
-        toInsert.prev = null;
+        toInsert.prev = &root;
```

Comment:
I would prefer if we can live without double-pointers as they make reading the code pretty hard (especially if only `prev` is a double-pointer, while `next` isn't). For example this looks suspicious:

```d
toInsert.prev = &root;
...
root = toInsert; // doesn't toInsert.prev point to itself now?
```